src/main/java/com/cmlteam/serv/Serv.java

```diff
@@ -153,4 +115,72 @@ private static IllegalStateException printHelpAndExit(String message, Options op
     System.exit(1);
     return new IllegalStateException();
   }
+
+  private static String getOutputStringForMultipleFilesDownload(String urlRoot) {
```

Comment:
Please let's extract the whole help message generation logic into separate class to make code less cluttered.

Reply:
resolved